kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KroxyliciousSteps.java

```diff
@@ -31,18 +31,70 @@ public static void produceMessages(String namespace, String topicName, String bo
         KafkaUtils.produceMessageWithTestClients(namespace, topicName, bootstrap, message, numberOfMessages);
     }
 
+    /**
+     * Produce single message with kcat.
+     *
+     * @param namespace the namespace
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param message the message
+     */
+    public static void produceSingleMessageWithKcat(String namespace, String topicName, String bootstrap, String message) {
+        KafkaUtils.produceSingleMessageWithKcat(namespace, topicName, bootstrap, message);
+    }
+
+    /**
+     * Produce messages with kafka go.
+     *
+     * @param namespace the namespace
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     */
+    public static void produceMessagesWithKafkaGo(String namespace, String topicName, String bootstrap, String message) {
+        KafkaUtils.produceSingleMessageWithKafkaGo(namespace, topicName, bootstrap, message);
+    }
+
     /**
      * Consume messages.
      *
      * @param namespace the namespace
      * @param topicName the topic name
      * @param bootstrap the bootstrap
+     * @param message the message
      * @param numberOfMessages the number of messages
      * @param timeout the timeout
      * @return the string
      */
-    public static String consumeMessages(String namespace, String topicName, String bootstrap, int numberOfMessages, Duration timeout) {
-        return KafkaUtils.consumeMessageWithTestClients(namespace, topicName, bootstrap, numberOfMessages, timeout);
+    public static String consumeMessages(String namespace, String topicName, String bootstrap, String message, int numberOfMessages, Duration timeout) {
+        return KafkaUtils.consumeMessageWithTestClients(namespace, topicName, bootstrap, message, numberOfMessages, timeout);
+    }
+
+    /**
+     * Consume messages with kcat
+     *
+     * @param namespace the namespace
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param message the message
+     * @param timeout the timeout
+     * @return the string
+     */
+    public static String consumeMessagesWithKcat(String namespace, String topicName, String bootstrap, String message, Duration timeout) {
+        return KafkaUtils.consumeMessagesWithKcat(namespace, topicName, bootstrap, message, timeout);
+    }
+
+    /**
+     * Consume messages with kafka go
```

Comment:
```suggestion
     * Consume messages with kaf (sarama client based CLI)
```

let's be clear, we are integrating with a specific CLI, not Go lang as a whole.

Reply:
sure. Thanks!